From c26b8c4e4a9b86d9751782ee0d5f4373da736e3c Mon Sep 17 00:00:00 2001
From: Greg Hackmann <ghackmann@google.com>
Date: Mon, 11 Nov 2013 14:31:12 -0800
Subject: [PATCH] video: adf: fix ADF_MAX_ATTACHMENTS declaration

Userspace-facing ADF_MAX_ATTACHMENTS must be in terms of
userspace-facing struct adf_attachment_config

Change-Id: Iaaddcd6366f13b3e52eb3911efcfff8a61e0b225
Signed-off-by: Greg Hackmann <ghackmann@google.com>
---
 include/uapi/video/adf.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/uapi/video/adf.h b/include/uapi/video/adf.h
index 73cb936..38458f6 100644
--- a/include/uapi/video/adf.h
+++ b/include/uapi/video/adf.h
@@ -233,7 +233,7 @@ struct adf_device_data {
 	size_t custom_data_size;
 	void __user *custom_data;
 };
-#define ADF_MAX_ATTACHMENTS (4096 / sizeof(struct adf_attachment))
+#define ADF_MAX_ATTACHMENTS (4096 / sizeof(struct adf_attachment_config))
 
 /**
  * struct adf_device_data - describes a display interface
-- 
1.8.0

